domain/schema/model/triggers/user-public-ssh-key.gen.go

```diff
@@ -0,0 +1,48 @@
+// Code generated by triggergen. DO NOT EDIT.
+
+package triggers
+
+import (
+	"fmt"
+
+	"github.com/juju/juju/core/database/schema"
+)
+
+
+// ChangeLogTriggersForUserPublicSshKey generates the triggers for the 
+// user_public_ssh_key table.
+func ChangeLogTriggersForUserPublicSshKey(columnName string, namespaceID int) func() schema.Patch {
```

Comment:
```suggestion
func ChangeLogTriggersForUserPublicSSHKey(columnName string, namespaceID int) func() schema.Patch {
```

It is not the onomatopoeic word. Can you possible fix the code generator to correct this?

Reply:
I don't think we can.